### ISSUE TITLE
Proper case

### ISSUE THREAD

**Issue by jonnew** [open-ephys/oe-acquisition-board-gen3#1]
The case at least needs a bottom to protect the PCB from shorts and water. I would recommend we consider switching to an aluminum case with light pipes for fit-and-finish.

**Comment by jvoigts** [comment]
I agree that a proper case would be nice, but I feel like given that we have a backlog of things to get out, the time required to set this up would likely be better spent on other projects? E.g. onix & miniscopes? What about just adding a thin laser cut acrylic sheet to the bottom? This way we could use the same existing cases etc.

**Comment by jonnew** [comment]
Sounds good as long as it covers the bottom.

**Comment by filcarv** [comment]
I believe that an acrylic sheet may require a redesign of the 3D case because the FPGA module is very tight. Any extra mm will require 3D redesign.

**Comment by jonnew** [comment]
Another request: The backlighting of the symbols showing digital in analog in etc need to be fixed so they can be read.
